toolkit/scss/forms/_questions.scss

```diff
@@ -1,16 +1,35 @@
-@import "_colours";
-@import "_typography.scss";
+@import "colours";
+@import "typography";
 @import "shared_scss/_dmspeak.scss";
 @import "shared_placeholders/_visuallyhidden.scss";
 
+
+// Spacing rules based on those found in govuk elements selection buttons
```

Comment:
It might be good to move this into its own placeholder file, just incase we end up wanting to import `_questions.scss` into anything in future (in which case the selectors produced will appear in all those places).